docs/swarm-mode-auto.md

```diff
@@ -84,6 +84,9 @@ docker service create --name proxy \
 
 We opened the ports *80* and *443*. External requests will be routed through them towards destination services. The proxy is attached to the *proxy* network and has the mode set to *swarm*. The proxy must belong to the same network as the listener. They will exchange information whenever a service is created or removed.
 
+!!! info
+	If you call the service something other than *proxy*, like in the example above, you have to pass the environment variable `SERVICE_NAME` on creation. The value of `SERVICE_NAME` has to be the same as the name of the service.
```

Comment:
Yeah, I can see that now. I guess a simple rephrasing of the first part of the sentence would eliminate the need for that form of clarification.

How about this? Shorter and to the point.
"If you name the service something other than *proxy*, you have to pass the environment variable ..."